src/app/swap/page.tsx

```diff
@@ -170,30 +175,37 @@ export default function Swap() {
         tokenOut: NetworkToken,
         amountIn: string
       ) => {
-        const data = {
-          tokenIn: tokenIn.defuse_asset_id,
-          tokenOut: tokenOut.defuse_asset_id,
-          amountIn: parseUnits(amountIn, tokenIn?.decimals ?? 0).toString(),
-        }
-        const { bestEstimate } = await getSwapEstimateBot(data)
-        if (bestEstimate === null) {
+        try {
+          const data = {
+            tokenIn: tokenIn.defuse_asset_id,
+            tokenOut: tokenOut.defuse_asset_id,
+            amountIn: parseUnits(amountIn, tokenIn?.decimals ?? 0).toString(),
+          }
+          const { bestEstimate } = await getSwapEstimateBot(data)
+          if (bestEstimate === null) {
+            isProgrammaticUpdate.current = true
+            setValue("tokenOut", "")
+            setErrorMsg(ErrorEnum.NO_QUOTES)
```

Comment:
Adds new No Quotes err

Reply:
`setErrorMsg(ErrorEnum.NO_QUOTES)` is redundant, we should use `react-hook-form` in right way otherwise there is no point in this library.

Reply:
This can't be handled through react-hook-form , as this a response from quote